models/member.ini

```diff
@@ -10,6 +10,7 @@ type = string
 [fields.level]
 label = Level
 type = select
+;source = site.query('/contributing/membership')
 choices = platinum, gold, silver, bronze, adhoc, professional, individual, inkind
 choice_labels = Platinum, Gold, Silver, Bronze, Professional, Individual, Ad hoc, In kind
```

Comment:
Ad Hoc sponsors are companies that have given donations that aren't formally part of a membership deal, but aren't just donations of services of the company itself (e.g., StickerMule giving us stickers, CircleCI giving us extra credits).